codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSHTTPBindingProtocolGenerator.kt

```diff
@@ -50,7 +49,6 @@ abstract class AWSHTTPBindingProtocolGenerator(
             customizations,
             operationMiddleware,
             getProtocolHttpBindingResolver(ctx, defaultContentType),
-            imports,
```

Comment:
`imports` is no longer a param since imports are now managed by writing symbols to SwiftWriter.